_includes/sections/dns.html

```diff
@@ -285,6 +285,22 @@ <h4>Terms</h4>
     <li>DNSCrypt - An older yet robust method of encrypting DNS.</li>
   </ul>
 
+  <h4>How to Validate</h4>
```

Comment:
```suggestion
  <h4>How to verify DNS is encrypted</h4>
```

This is not a perfect suggestion either, but "Validate" is a bit unclear on what is being validated.